### PR TITLE
[CI] Fix update slow tests

### PR DESCRIPTION
**Pull request by clee2000**
* Add pytorchbot to list of approvers for file
* Add labels to the auto created PR

The auto generated PR is currently not merging due to some failing tests on slow workflow that were supposed to be moved back to normal

idk if this has much value, clearly we've been managing without the update

### CODE REVIEW

File: .github/merge_rules.yaml [hunk 1]
```diff
@@ -86,6 +86,17 @@
...
+- name: OSS CI / pytorchbot / slow tests
+  patterns:
+  - test/slow_tests.json
+  approved_by:
+  - pytorchbot
+  ignore_flaky_failures: false
+  mandatory_checks_name:
+  - EasyCLA
+  - Lint
+  - pull
```
Comment (huydhn) [inline .github/merge_rules.yaml:98]: Also `- slow`?

File: tools/testing/update_slow_tests.py [hunk 2]
```diff
@@ -204,5 +213,6 @@ def search_for_open_pr(
         # no existing pr, so make a new one and approve it
         pr_num = make_pr("pytorch/pytorch", params)
         time.sleep(5)
+        add_labels("pytorch/pytorch", pr_num, ["ciflow/slow", "ci-no-td"])
```
Comment (huydhn) [inline tools/testing/update_slow_tests.py:216]: Alternatively, you could update `titleRegexToLabel` in https://github.com/pytorch/test-infra/blob/main/torchci/lib/bot/autoLabelBot.ts#L19 to add these labels.  I think that would be cleaner than writing another `add_labels` function here

File: .github/merge_rules.yaml (hunk 1 above)
Comment (ZainRizvi) [reply .github/merge_rules.yaml:98]: Any idea why trunk  isn't part of most of these rules?

**Review by ZainRizvi — APPROVED** [review]

File: .github/merge_rules.yaml (hunk 1 above)
Comment (clee2000) [reply .github/merge_rules.yaml:98]: Not sure

File: tools/testing/update_slow_tests.py (hunk 2 above)
Comment (clee2000) [reply tools/testing/update_slow_tests.py:216]: I think I'd prefer to leave it here in order to keep everything in one place

File: .github/merge_rules.yaml (hunk 1 above)
Comment (huydhn) [reply .github/merge_rules.yaml:98]: I think trunk used to mean only jobs running in trunk, not PR.  Then we make it running as part of merge process, so it's not here.

**Pull request closed without merging**